Fix #1224: Filtering of operations in operation list by activation ID… (#1226)
wultra/powerauth-server: docs/WebServices-Methods.md

|-----------|-----------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| `String`  | `userId`        | The identifier of the user                                                                                                                                                      |
| `String`  | `applicationId` | The identifier of the application                                                                                                                                               |

> | `String`  | `activationId`  | Optional. An identifier of an activation paired with requested operation/s                                                                                                      |

Reviewer: ```suggestion
| `String`  | `activationId`  | Optional. An identifier of activation when the list is requested on a particular device                                                                                                      |
```

|-----------|-----------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| `String`  | `userId`        | The identifier of the user                                                                                                                                                      |
| `String`  | `applicationId` | The identifier of the application                                                                                                                                               |

> | `String`  | `activationId`  | Optional. An identifier of an activation paired with requested operation/s                                                                                                      |

Reviewer: See above
